dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/MessageConversationController.java

```diff
@@ -414,7 +486,7 @@ public void postMessageConversationFeedback( @RequestParam( "subject" ) String s
             return responseNode;
         }
 
-        if ( !configurationService.isUserInFeedbackRecipientUserGroup( userToAssign ) )
+        if ( messageConversation.getMessageType() == MessageType.TICKET && !configurationService.isUserInFeedbackRecipientUserGroup( userToAssign ) )
```

Comment:
OR messageType is Validation result

Reply:
Why? Isn't it just for tickets that we look at the feedback recipient group?